client/src/components/Twilio/index.js

```diff
@@ -0,0 +1,22 @@
+require('dotenv').config();
+
+const accountSid = process.env.TWILIO_ACCOUNT_SID;
+const authToken = process.env.TWILIO_AUTH_TOKEN;
+
+const gameMaker = require('twilio')(cfg.accountSid, cfg.authToken);
+
+//, "+19192745942", "+19195235455", "+19196968557
+
+var inviteeArray = ["+19192703244"];
```

Comment:
@MandyTorrence I'm guessing we'll want to initialize this as an empty array?

```suggestion
// number format "+19192703244"
var inviteeArray = [];
```